plio/static/plio/docs/download_csv_README.md

```diff
@@ -0,0 +1,46 @@
+# Plio Data
+
+This folder contains the following files:
+
+- `plio-meta-details`: contains the meta information for the plio:
+
+  - `id`: the unique identifier for the plio
+  - `name`: the name of the plio as set by you
+  - `video`: the URL of the video linked to the plio
+
+- `plio-interaction-details.csv`: contains the details of each interaction (referred to as `item` in Plio) added to the plio. The columns represent the following:
+
+  - `item_id`: the unique identifier for the interaction item
+  - `item_type`: the type of the interaction
+  - `item_time`: the time in the video when the interaction would appear
+
+  If the `item_type` is `question`, the following columns are also present:
+
+  - `question_type`: the type of question (e.g. `mcq`)
+  - `question_text`: the text of the question itself
+  - `question_options`: the options for the question
+  - `question_correct_answer`: the correct answer for the given question
```

Comment:
Will this be the correct answer ka text or the # of option which is correct? And we should also add a line saying that right?
for example if someone had options `1`, `2`, `3`, `4` then there will be a lot of confusion here